src/NServiceBus.Persistence.ServiceFabric.Tests/ComponentTests/Sagas/When_multiple_workers_retrieve_same_saga_on_different_threads.cs

```diff
@@ -33,7 +33,9 @@ public class When_multiple_workers_retrieve_same_saga_on_different_threads : Sag
             {
                 var winningContext = configuration.GetContextBagForSagaStorage();
                 var winningSaveSession = await configuration.SynchronizedStorage.OpenSession(winningContext);
+                SetActiveSagaInstance(winningContext, new TestSaga(), new TestSagaData { Id = generatedSagaId, SomeId = correlationPropertyData });
```

Comment:
Now doing this call before the `Get `with some dummy data and after the `Get `with the real data to simulate better the core and so that the ActiveSagaInstance contains refreshed data.